Tests/AK/TestFlyString.cpp

```diff
@@ -68,15 +68,15 @@ TEST_CASE(long_string)
 
 TEST_CASE(from_string_view)
 {
-    auto fly1 = MUST(FlyString::from_utf8("thisisdefinitelymorethan7bytes"sv));
+    auto fly1 = MUST(Fly"thisisdefinitelymorethan7bytes"_string);
```

Comment:
Find and replace error, or weird new syntax?

Reply:
That's a find-replace error.

Reply:
yep, added `_fly_string` for this as well